Add error handler warning to update guide (#2841)
ibexa/documentation-developer: docs/infrastructure_and_maintenance/performance.md

- Always enable opcache for php-fpm/`mod_php`.
- Prefer php-fpm and web server using it over fast-cgi for lower overall memory usage.

!!! caution

>     If you are running [[= product_name =]] in version v4.6.8 and below, to avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, run the following commands:

Reviewer: ```suggestion
    To avoid deprecations when using PHP 8.2 or 8.3, v4.6.x needs an additional error handler. The error handler was added by default for projects that started with 4.6.8 or later. For projects that started on earlier versions, it needs do be added manually by running the command:
```
^^ sounds not very good language wise - main thing is that you might be on 4.6.8 or later now but just missed to add the error handler.

    The error handler is present by default for projects that started with 4.6.8 or later.
    For projects that started with earlier versions, you need to add the error handler manually by running the following commands:

    ``` bash
    composer config extra.runtime.error_handler "\\Ibexa\\Contracts\\Core\\MVC\\Symfony\\ErrorHandler\\Php82HideDeprecationsErrorHandler"
    composer dump-autoload
    ```

> 

Reviewer 2: It's clear that ["update_from_4.6"](https://ez-systems-developer-documentation--2841.com.readthedocs.build/en/2841/update_and_migration/from_4.6/update_from_4.6/) is only about 4.6 and in the continuation ["update_to_5.0" removes it](https://ez-systems-developer-documentation--2841.com.readthedocs.build/en/2841/update_and_migration/from_4.6/update_to_5.0/#update-ibexa-dxp-application).

But for "performance" when on 5.0 branch of our doc, this is less clear. It's not since 4.6 that this is needed, it's for 4.6 only. I would add something about 5.0 not needing this, or I would have it only on 4.6 branch but not on 5.0 branch.

- Always enable opcache for php-fpm/`mod_php`.
- Prefer php-fpm and web server using it over fast-cgi for lower overall memory usage.

!!! caution

>     To avoid deprecations when using PHP 8.2 or 8.3, [= product_name =] v4.6.x needs an additional error handler.

Reviewer 3: ```suggestion
    To avoid deprecations when using PHP 8.2 or 8.3, [[= product_name =]] v4.6 needs an additional error handler.
```

https://ez-systems-developer-documentation--2841.com.readthedocs.build/en/2841/infrastructure_and_maintenance/performance/#web-server is not rendered correctly

    The error handler is present by default for projects that started with 4.6.8 or later.

>     For projects that started with earlier versions, you need to added the error handler manually by running the following commands:

Reviewer: ```suggestion
    For projects that started with earlier versions, you need to add the error handler manually by running the following commands:
```

    ``` bash
    composer config extra.runtime.error_handler "\\Ibexa\\Contracts\\Core\\MVC\\Symfony\\ErrorHandler\\Php82HideDeprecationsErrorHandler"
    composer dump-autoload
    ```

> 

Author: ```suggestion
```

>     In newer versions of [[= product_name =]], the error handler is added by default.

Reviewer 3: ```suggestion
```
This repeats the message earlier, I don't think it's needed - also taking into account that "newer versions" is ambigious - it's not sure if it includes v5, and v5 does not need this error handler (it's completely removed).
